tools/clang/unittests/HLSL/CMakeLists.txt

```diff
@@ -10,6 +10,8 @@ set( LLVM_LINK_COMPONENTS
   dxcsupport
   hlsl
   option
+  bitwriter
```

Comment:
Taking a dependency on DxilContainerWriter in DxilContainerAssembler.obj pulled additional dependencies in, leading to these libs being required.